draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1191,9 +1191,9 @@ Inputs:
 
   chain:
   : An array of zero or more base64 encoded CA certificates. The first element
-    is the signer of the `submission`; the second certifies the first; etc.
+    is the certifier of the `submission`; the second certifies the first; etc.
     The last element of `chain` (or, if `chain` is an empty array, the
-    `submission`) either is, or is certified by, an accepted trust anchor.
+    `submission`) is certified by an accepted trust anchor.
```

Comment:
Not sure I recall - why are we no longer allowing inclusion of the trust anchor itself in the submission?